fluentd/fluent.conf

```diff
@@ -135,18 +134,36 @@
     bearer_token_file /var/run/secrets/kubernetes.io/serviceaccount/token
     ca_file /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
   </filter>
+  <filter **kibana**>
+    type record_transformer
+    enable_ruby
+    <record>
+      log ${err || msg}
+    </record>
+    remove_keys req,res,msg,name,level,v,pid,err
+  </filter>
   <filter kubernetes.**>
     type flatten_hash
     separator _
   </filter>
+  <filter kubernetes.**>
+    type record_transformer
+    enable_ruby
+    <record>
+      hostname ${kubernetes_host}
+      message ${log}
+      @version 3.1
```

Comment:
OK, so it's kind of arbitrary. This is a totally minor nit but can it track
Origin's version? Since it's kinda upstream.

On Tue, Sep 29, 2015 at 1:52 PM, Eric Wolinetz <notifications@github.com>
wrote:

> In fluentd/fluent.conf
> <https://github.com/openshift/origin-aggregated-logging/pull/4#discussion_r40704504>
> :
>
> >    <filter kubernetes.**>
> >      type flatten_hash
> >      separator _
> >    </filter>
> > +  <filter kubernetes.**>
> > +    type record_transformer
> > +    enable_ruby
> > +    <record>
> > +      hostname ${kubernetes_host}
> > +      message ${log}
> > +      @version 3.1
>
> This is part of the ECL initiative. The idea for this is that if we make
> further fluentd config changes we would increment this version so that if
> there is an ES instance that persists multiple fluentd config versions we
> could use the @version <https://github.com/version> tag so that we
> correlate data between versions (in the chance that there is no backwards
> compat).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/4/files#r40704504>
> .
>